package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "substr-polyfill",
-  "version": "0.0.2",
-  "description": "Modular polyfill for substr(-1) for IE browsers.  Respectfully copied from MDN and added to a module.",
+  "version": "0.0.3",
```

Comment:
no need to bump package in PR. I'll adjust it. nbd.

Reply:
cool. no worries.